app/lib/bk/compat/parsers/bitrise/github_status.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+module BK
+  module Compat
+    module BitriseSteps
+      # Handles the translation of github-status
+      class Translator
+        def translate_github_status(inputs)
+          context = inputs['context'] || inputs['status_identifier'] || 'continuous-integration/buildkite'
+          warnings = generate_warnings_for_unsupported_options(inputs.keys)
+
+          github_status = {
+            'context' => context
+          }
+
+          step = BK::Compat::CommandStep.new
+
+          step.notify = [{ 'github_commit_status' => github_status }]
+          step.add_commands(warnings) unless warnings.empty?
+
+          step
+        end
```

Comment:
You can create the command object with the corresponding elements instead of assigning them and then invoking the value for returning:

```suggestion
          BK::Compat::CommandStep.new(
            notify: [{ 'github_commit_status' => { 'context' => context } }]
            commands: [warnings].compact
          )
```